src/components/Link/Link.tsx

```diff
@@ -87,8 +89,8 @@ export const Link = <T extends ElementType = "a">({
       <IconWrapper $size={size}>
         <Icon
           name="popout"
-          className="external-icon"
-          data-testid="external-icon"
+          className={icon ?? "external-icon"}
```

Comment:
i am not sure i understand how this should work. we pass the icon name as a class but the icon remains always "popout" ?

Reply:
Sorry missed the place I should have added in the name too

Reply:
Fixed it